app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -114,6 +114,10 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
         userIcon = findViewById(R.id.userIcon)
     }
 
+    private fun updateFragment() {
+        val fragment = supportFragmentManager.findFragmentByTag(FRAGMENT_TAG)!! as UserPlaylistsFragment
+        fragment.notifyUserChanged()
+    }
```

Comment:
I don't see where use assigns the tag to a `UserPlaylistsFragment`. Was the tag meant to be used on line 70?

Also, why looking for the fragment every time and not put it in a value?

Reply:
You are correct, I forgot to assign the tag during initialization. The reason we are using tags instead of putting fragment in a value is that the Fragment manager takes a fragment class as input, and takes care of building its own instance, so we do not have access to the instance.

I still think this is cleaner that inflating a recyclerview instead of using the fragmentManager to replace a RecyclerViewContainer because it leaves all the fragment lifecycle handling to be correctly handled by the fragmentManager

If you know of a cleaner way, I will gladly implement it ;)

Reply:
OK I see why you do that now 👍🏻